module/VuFind/src/VuFind/Db/Service/LoginTokenService.php

```diff
@@ -148,7 +185,24 @@ public function deleteByUser(UserEntityInterface|int $userOrId): void
     public function getByUser(UserEntityInterface|int $userOrId, bool $grouped = true): array
     {
         $userId = is_int($userOrId) ? $userOrId : $userOrId->getId();
-        return $this->getDbTable('LoginToken')->getByUserId($userId, $grouped);
+        $dql = 'SELECT lt '
+            . 'FROM ' . $this->getEntityClass(LoginTokenEntityInterface::class) . ' lt '
+            . 'WHERE lt.userId = :userId '
```

Comment:
I turned on persistent logins, logged in, and then went to my profile page to look at my token. That caused a failure with this error:

```
[Semantical Error] line 0, col 131 near 'userId = :userId': Error: Class VuFind\Db\Entity\LoginToken has no field or association named userId
```